config/services.yml

```diff
@@ -513,6 +550,43 @@ https://cmr.uat.earthdata.nasa.gov:
           STAGING_PATH: public/podaac/netcdf-converter
     collections:
       - C1234410736-POCLOUD
+      - C1215664073-GES_DISC
```

Comment:
Also note that this looks like the same set of collections configured above for the ASF GDAL subsetter. Since the capabilities of this subsetter (convert to TIFF) are already supported by the ASF GDAL subsetter, it is not possible to construct a harmony request that would execute this service. Harmony matches a request to a backend service by finding the first service from top to bottom of the services.yml file, and so a request for tiff format will always result in using the ASF GDAL subsetter.

Reply:
Jay and I talked - they will talk to PODAAC about whether the COG converter is ready for testing for their team. They'll also use a different set of collections to test some with the ASF subsetter and others with the COG converter if it's ready for use to test in UAT.

Reply:
Thanks Chris. Talked to Frank Greguska and have updated the collections. Please merge.